ztests/suite/zqd/worker-count.yaml

```diff
@@ -15,14 +15,13 @@ script: |
   # it does not matter which one.
   #
   ZNG_FILE_PATH=$(find . -name "d-*.zng" -print | head -1)
-  CHUNK_INFO=$(basename $ZNG_FILE_PATH)
   #
   # The count() from zq should be identical to 
   # the count() from zapi get -chunk
   #
   zq -t "count()" $ZNG_FILE_PATH > zqcount.tzng
   zapi -h $ZQD_HOST -s testsp get -t \
-    -chunk $CHUNK_INFO "count()" > zapicount.tzng
```

Comment:
This change also happened in 1502, since I pass in the full path of the chunk rather than just the basename.

Reply:
I have this weird feeling that last night we were doing "parallel programming" (in the same sense as "parallel play") in a lot of the same files. :-)